recipes-images/images/engicam-gwc.bb

```diff
@@ -105,6 +105,9 @@ IMAGE_INSTALL:append = " \
 	libedit \
 	gpsd \
 	libgps \
+        libestr \
+        libfastjson \
```

Comment:
Are `libestr` and `libfastjson` needed? I see them in rsyslog dependencies:
```
DEPENDS = "zlib libestr libfastjson bison-native flex-native liblogging"
```
So should not be needed. Did a small test and can confirm.

(so with the below comment we could remove the whole modifications to `recipes-images/images/engicam-gwc.bb` in this PR)

Reply:
I've explicitly added them since when running `bitbake rsyslog`  It does build `rsyslog*.rpm` but not all it's dependencies, I had to run bitbake for both `libfastjson` and `libestr` to get the rpms.

But if setting like this should be fine, then it's better to removed them for sure